src/synapse/api.py

```diff
@@ -18,11 +18,31 @@
 from urllib3.util import Retry
 
 from charm_state import CharmState
-from constants import MJOLNIR_MANAGEMENT_ROOM, MJOLNIR_MEMBERSHIP_ROOM, SYNAPSE_URL
 from user import User
 
 logger = logging.getLogger(__name__)
 
+CHECK_ALIVE_NAME = "synapse-alive"
```

Comment:
I think this is only used in  `pebble.py` can you move it there and make sure there are no more constants defined here that are only used in one file different from this?

Reply:
I just checked and it's also used in `workflow.py` : https://github.com/canonical/synapse-operator/blob/221a25a0f91b0a62a642979a39b55e6165c47406/src/synapse/workload.py#L94 but maybe the function that is using it is not in the right place as it's also related to pebble.

What do you think about this? should I keep the constant in `api.py` or move it to `pebble.py` together with the `check_alive()` function?

I also did a scan of the constants added to `api.py` and made sure that they are all referenced in more than 2 files.

Reply:
It'd be probably better to define the in `workload.py` and export it in the `src/synapse/__init__.py` to keep it more cohesive.

Reply:
Will do that, thanks.

Reply:
I imported all constants in `src/synapse/__init__.py` and changed all use of constants in the charm to add the `synapse.` prefix